### ISSUE TITLE
[prefer-expect-assertions] incorrect report when using multiple describes

### ISSUE THREAD

**Issue by WhoAmIRUS** [jest-community/eslint-plugin-jest#1088]
**Rule settings:**

![image](https://user-images.githubusercontent.com/29947304/164719099-fe02c15c-0cdf-469f-8aa1-a4b60ecbb65a.png)

I have this eslint error with nested describe. Is it right rule behavior?

![image](https://user-images.githubusercontent.com/29947304/164718632-647621b6-a040-42ef-89b6-ea362b34f1e9.png)

Hide test:
```
test.each<{ result: string; args: FormatNumberByTypeCommonArguments }>([
      { result: '11 ₽/день', args: {} },
      { result: '11 $/день', args: { currency: 'USD' } },
    ])('without fixed numerator currency value', ({ result, args }) => {
      expect(formatNumberByType(11, { type: 'ratio', numerator: 'currency', denominator: 'day' }, args)).toBe(result);
    });
```

If I remove the first test.each, I will have no error.

**Comment by G-Rath** [comment]
Yeah this is a bug.

Just for future, could you please use code blocks of details like your code and config rather than just images?

**Comment by WhoAmIRUS** [comment]
Yea, sorry, I will. Thanck you for fix!
